src/main/java/com/adobe/cq/commerce/graphql/client/GraphqlClientConfiguration.java

```diff
@@ -73,19 +73,23 @@
 
     @AttributeDefinition(
         name = "Http connection timeout",
-        description = "The timeout in milliseconds until a connection is established. A timeout value of zero is interpreted as an infinite timeout.",
+        description = "The timeout in milliseconds until a connection is established. Is the timeout longer than the default timeout a"
+            + " warning will be logged. Is it 0 or smaller the configured service will not start.",
```

Comment:
Would be nice to name the default values in the description, since the AEM config UI doesn't show them.

Reply:
Good catch, fixed.